benchmarks/preset_configs/bert/gpu/scripted_batch_1.json

```diff
@@ -0,0 +1,14 @@
+{
+  "url": "https://torchserve.s3.amazonaws.com/mar_files/BERTSeqClassification_Torchscript_batch.mar",
```

Comment:
Please check the model urls for GPU bench-marking - at present we have separate BERT models for CPU and GPU

Reply:
Fixed.